### PR TITLE
DA-139 - Boxbot tutorial

### PR DESCRIPTION
**Pull request by mcvella**

### CODE REVIEW

File: docs/tutorials/build-an-outdoor-rover.md [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+tags: ["base", "outdoor", "solar", "rover"]
+description: "How to build a very capable outdoor rover and control it with Viam."
+# SME: Matt Vella
+---
+
+Approaching robotics can be daunting.
+Many of us who have tried have ended up with empty wallets, life to catch up on, and spare parts in our closets.
+Some of us have had some success starting with robotic kits or learning robots.
+You know - the ones that are fairly easy to get up and running, but ultimately are expensive toys?
+In my humble opinion, these robots are somewhat missing the point: people want to create robotics *that* *do real things, that solve real life problems*.
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:15]:
```suggestion
In my humble opinion, these robots are somewhat missing the point: people want to create robots *that* *do real things, that solve real-life problems*.
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+What to do?
+Well, you *can* certainly purchase a very capable outdoor robotic rover for upwards of 12 thousand dollars.
+It will still be hard to learn how to expand and program it, but it's at least capable, right?
+
+<div class="embed-responsive embed-responsive-16by9">
+<iframe width="415" height="737" src="https://www.youtube.com/embed/9OzwozzckdY" title="BoxBot" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+</div>
+
+Thankfully I work at Viam, where we are building a platform that makes robotics and robotic automation approachable, and therefore much easier to build useful robots.
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:24]:
```suggestion
But at Viam we are building a platform that makes robotics and robotic automation approachable, therefore it is much easier to build useful robots.
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+Now, when I am an employee at *any* company one of the most important things is that I truly believe in the product.
+So, I challenged myself with the following proof: *"Can I build a weatherproof, outdoor rover that is capable of doing useful things for under \$500?"*
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:26]:
```suggestion
So, I challenged myself with the following proof: *"Can I build a weatherproof, outdoor rover that is capable of doing useful things for under $500?"*
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 4]
```diff
@@ -0,0 +1,304 @@
...
+
+## Overall Boxbot design
+
+The success of this project was based on two key factors:
+
+1. Hoverboard hub motors are sealed and waterproof, can carry a good payload, are capable of navigating outdoor terrain like gravel and grass, and are somewhat readily available.
+
+2. Pelican-style plastic boxes are not only able to protect your electronics against the elements, but are also sturdy enough for hub motors to be directly bolted on, and able to carry additional payload on top.
+
+Regardless of whatever real-life things you might have in mind for your outdoor robotic base, these parts are required for this design.
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:38]:
```suggestion
Regardless of the materials you might have in mind for your outdoor robotic base, these parts are required for this design.
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 5]
```diff
@@ -0,0 +1,304 @@
...
+Other advantages of using a case of this type is that you can easily open and close it to make adjustments, and you can conveniently carry the entire robot with the case's handle.
+
+## Parts list for your Boxbot
+
+Some of the equipment listed here is optional.
+For example, you might not need a night-vision camera if you won't operate it at night and you will not need a solar panel and charge controller if you plan on using your rover occasionally and charging it in-between.
+
+|Part | Quantity | Price (each) | Price total | Notes |
+| --- | --- | --- | --- | --- |
+| [Weatherproof plastic case](https://www.alibaba.com/product-detail/Waterproof-Foam-Pu-eva-epe-xpe_1600097495864.html) | 1 | $30.50 | $30.50 | This is the key to protecting your electronics and holding the robot together. Make sure it is thick, strong plastic. Also, you'll need to get one that will fit the solar panel being mounted on top. Finally, get one with adjustable foam - this is very handy for cushioning your electronic components inside.|
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:48]:
```suggestion
| [Weatherproof plastic case](https://www.alibaba.com/product-detail/Waterproof-Foam-Pu-eva-epe-xpe_1600097495864.html) | 1 | $30.50 | $30.50 | This is the key to protecting your electronics and holding the robot together. Make sure it is a thick, strong plastic and can fit the solar panel mounted on top. Finally, get one with adjustable foam inside - this is very handy for cushioning your electronic components.|
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 6]
```diff
@@ -0,0 +1,304 @@
...
+| [Night vision camera](https://www.amazon.com/gp/product/B07C1N9R4Z)|1|$40|$40|Optional, but a webcam of some sort is highly recommended.|
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:49]:
```suggestion
| [Night vision camera](https://www.amazon.com/gp/product/B07C1N9R4Z)|1|$40|$40|Night vision one is optional, but a webcam of some sort is highly recommended.|
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 7]
```diff
@@ -0,0 +1,304 @@
...
+| Raspberry Pi 3B or 4B with empty microSD card|1|$100|$100|Due to supply shortages, Raspberry Pi prices are fluctuating dramatically.|
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:50]:
```suggestion
| Raspberry Pi 3B or 4B with microSD card|1|$100|$100|Note: Due to supply shortages, Raspberry Pi prices are fluctuating dramatically.|
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 8]
```diff
@@ -0,0 +1,304 @@
...
+| [12V Battery](https://www.amazon.com/LiFePO4-Battery-Miady-Rechargeable-Maintenance-Free/dp/B089VXSBC6)|1|$65|$65|Lots of options here - some motors might run better on 24v. You can use a battery with less storage if you don't plan to run it continuously.|
+| [Brushless hub motor wheels](https://www.alibaba.com/product-detail/Electric-Wheel-Hoverboard-DC-Hub-6_60615157026.html)|2|$30|$60|Better yet - source them from a used kids' [hoverboard](https://www.amazon.com/RIDE-SWFT-Hoverboard-Balancing-Front-Facing/dp/B08N5DSVY3) (this is what I did).|
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:52]:
```suggestion
| [Brushless hub motor wheels](https://www.alibaba.com/product-detail/Electric-Wheel-Hoverboard-DC-Hub-6_60615157026.html)|2|$30|$60|Better yet - source them from a used [hoverboard](https://www.amazon.com/RIDE-SWFT-Hoverboard-Balancing-Front-Facing/dp/B08N5DSVY3) (this is what I did, sorry son).|
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 9]
```diff
@@ -0,0 +1,304 @@
...
+|[GPS module](https://www.amazon.com/Receiver-Antenna-Gmouse-Laptop-Navigation/dp/B073P3Y48Q/)|1|$20|$20|optional|
+|[Solar charge controller](https://www.amazon.com/Renogy-Wanderer-Amp-12V-24V/dp/B07NPDWZJ7)|1|$20|$20|optional|
+|[25w Solar panel](https://www.alibaba.com/product-detail/High-Efficiency-25W-Polycrystalline-Crystalline-Solar_60814369754.html)|1|$10|$10|optional|
+[12V to 5V DC USB Type-C Right Angle Step-Down Power Converter](https://www.amazon.com/gp/product/B086KTGRH1/)|1|$13.50|$13.50|To power the pi from a 12v battery.|
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:56]:
```suggestion
[12V to 5V DC USB Type-C Right Angle Step-Down Power Converter](https://www.amazon.com/gp/product/B086KTGRH1/)|1|$13.50|$13.50|To power the Pi from a 12v battery.|
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 10]
```diff
@@ -0,0 +1,304 @@
...
+|[Brushless motor controller](https://www.amazon.com/RioRand-6-60V-Brushless-Electric-Controller/dp/B087M2378D)|2|$18|$36|
+|[Caster wheel](https://www.harborfreight.com/3-inch-x-3-4-quarter-inch-stem-swivel-caster-90997.html)|2|$5|$10|
+|Galvanized nuts/bolts, water sealing epoxy or caulk, washers, misc wires, brackets, etc|1|$15|$15|Bolts are for mounting your hub motors and other components. I used a bracket I had lying in my basement plus some bracket/rails from my kids erector set to mount the solar panel.|
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:59]:
```suggestion
|Galvanized nuts/bolts, water sealing epoxy or caulk, washers, misc wires, brackets, etc|1|$15|$15|Bolts are for mounting your hub motors and other components. I used a bracket I had lying in my basement plus some brackets/rails from my kids' erector set to mount the solar panel.|
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 11]
```diff
@@ -0,0 +1,304 @@
...
+
+For the hub motor mounts, drill two holes through each mount, line each one up, then drill through these holes into the bottom of the plastic case.
+While doing this I realized that I'd also want to run the wires from the hub motor into the case under the mount (to better protect the wires), so I drilled a smaller hole under where each of the mounts would be for that purpose.
+
+<div class="td-max-width-on-larger-screens">
+  <img src="../img/outdoor-rover-boxbot/mounted-hub-motor.png"  style="float:right;margin-left:1em;" alt="Mounted hub motor." title="Mounted hub motor." width="250" />
+</div>
+
+At this point, I want to mention that although the case is intrinsically waterproof, we'll want to make sure that we waterproof any holes that we make in the case.
+For this, I used a marine epoxy from the hardware store, surrounding where the bolts went though the case.
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:97]:
```suggestion
For this, I used marine epoxy from the hardware store, surrounding where the bolts went through the case.
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 12]
```diff
@@ -0,0 +1,304 @@
...
+</div>
+Try pushing your base around, but don't be too sad if it doesn't go perfectly straight; caster wheels don't exactly lend themselves to that (especially when not faced with constant forward force; you know this if you've ever let go of your shopping cart in the grocery store in the middle of a push).
+
+If you're feeling brave, try sitting on it (carefully!) to see if it can support your weight.
+This will depend wildly on your weight and the strength of the case you purchased; if you are worried you could try putting something else less heavy on top like a couple cinder blocks or barbells.
+
+Now could be a good time to attach a camera in a similar way to how you attached the wheels, but through the top of the case.
+Don't forget that wherever you decide to feed the camera wire into the case, you'll want to be sure you leave enough slack to open and close the case.
+I found that the rover handles much better with the hub motors in the back, acting as rear-wheel drive.
+If you are not sure if that's how you'll want to run yours, wait until after you've tested how it handles to mount a camera.
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:116]:
```suggestion
If you are not sure if that's how you'll want to run yours, wait until after you've tested how it handles, and then mount a camera.
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 13]
```diff
@@ -0,0 +1,304 @@
...
+
+## Wire it up
+
+OK, I'll be honest.
+I wanted to add an exclamation point here to motivate us - because in reality, this is the hardest part of the project.
+Specifically, setting up the motor controllers correctly is a bit of a challenge but I will do my best to describe how to best navigate them here.
+
+### Brushless motor controllers
+
+The first important note is that you'll need to be sure you're buying brushless motor controllers.
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:126]:
```suggestion
You have to make sure you're buying brushless motor controllers.
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 14]
```diff
@@ -0,0 +1,304 @@
...
+This is imperative because brushless DC motors (BDLC for short) have a tradeoff: while they and their controllers are more expensive and complicated to set up, they are more reliable and durable.
+Brushless motors do not rely on mechanical brushes that are prone to wear; instead they require an electronic communicator in the form of a more expensive and complex motor controller.
+
+So let's get to it.
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:130]:
```suggestion
Let's get to it.
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 15]
```diff
@@ -0,0 +1,304 @@
...
+Note that all of the steps in this section will need to be done twice, as we will be controlling two motors (one on each side of the base).
+The procedure for each side will look like:
+
+1. Solder motor controller
+2. Wire the hub motor to the motor controller
+3. Wire the battery to the motor controller
+4. Test controlling the motor with the motor controller potentiometer
+5. Wire to the raspberry pi
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:138]:
```suggestion
5. Wire to the Raspberry Pi
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 16]
```diff
@@ -0,0 +1,304 @@
...
+
+With the brushless motor controllers I used, you'll have to first solder a jumper where indicated in this diagram (label starts with "normally disconnected").
+This allows the pi to control the motor via PWM.
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:141]:
```suggestion
This allows the Pi to control the motor through PWM.
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 17]
```diff
@@ -0,0 +1,304 @@
...
+You'll also need to solder two pins to the controller just to the left of the white wire harness mount.
+(P.S. - A truly clutch solder tip for those of you who may not be very experienced with soldering: **heat the pin** with the soldering iron, **not the solder itself**! Once the pin is hot, touch the soldier to it. When someone taught me this it changed my life.)
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:143]:
```suggestion
(P.S. - A truly clutch solder tip for those of you who may not be very experienced with soldering: **heat the pin** with the soldering iron, **not the solder itself**! Once the pin is hot, touch the solder to it. When someone taught me this it changed my life.)
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 18]
```diff
@@ -0,0 +1,304 @@
...
+
+<div class="td-max-width-on-larger-screens">
+  <img src="../img/outdoor-rover-boxbot/motor-controller.jpg"  style="float:left;margin-right:1em;" alt="Brushless motor controller." title="Brushless motor controller." width="500" />
+</div>
+
+You'll then connect the motor phase A, B, and C wires from your hub motor to the left side of the controller, and the 5 hall sensor wires to the white wire harness on the bottom right.
+It is also worth mentioning that you may need to extend the wires from your hub motors if they are not long enough to work with.
+Likely, the color of the wires from your hub motor will match the diagram - at the very least the 5v and GND should.
+If any of the phase wire colors don't match, you might need to try swapping them when testing with the potentiometer.
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:152]:
```suggestion
If any of the phase wire colors don't match, you might need to swap them when testing with the potentiometer.
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 19]
```diff
@@ -0,0 +1,304 @@
...
+
+Otherwise you might have a half-built rover flying and crashing off of your desk/workspace - this could be bad.
+{{% /alert %}}
+
+If everything is now wired up and lifted, attaching the battery and turning the potentiometer with a screwdriver should power the motor and the wheel should spin.
+When you're done having fun with that, crank it back down until the motor is fully deactivated.
+
+### Organize the wires and components
+
+Now might be the time to figure out where you're laying everything out inside your case if you've not done so already.
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:172]:
```suggestion
Now might be the time to figure out where you're laying everything out inside your case.
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 20]
```diff
@@ -0,0 +1,304 @@
...
+You'll need a space for:
+
+<div class="td-max-width-on-larger-screens">
+  <img src="../img/outdoor-rover-boxbot/case-layout.jpg"  style="float:right;margin-left:1em;" alt="Brushless motor controller." title="Brushless motor controller." width="400" />
+</div>
+
+- Your already wired motor controllers
+- Your battery
+- Your pi
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:181]:
```suggestion
- Your Pi
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 21]
```diff
@@ -0,0 +1,304 @@
...
+- Your solar charge controller, if you're going to use solar charging
+
+As mentioned, having a case that comes with foam that can be easily cut/shaped is very helpful.
+Make a cutout for each of your components, considering how you are going to run your wires.
+You don't need to match my layout pictured here, in fact it could certainly be better organized.
+
+### Connect the motor controllers to the Pi
+
+Once that's sorted out, we're ready to connect 3 wires from the motor controller to the pi (you'll probably need a [pi pinout diagram](https://pinout.xyz) handy):
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:190]:
```suggestion
Once that's sorted out, we're ready to connect 3 wires from the motor controller to the Pi (you'll probably need a [Pi pinout diagram](https://pinout.xyz) handy):
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 22]
```diff
@@ -0,0 +1,304 @@
...
+
+1. One from the pin you added labeled **G** to any GND terminal on the pi
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:192]:
```suggestion
1. One from the pin you added labeled **G** to any GND terminal on the Pi
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 23]
```diff
@@ -0,0 +1,304 @@
...
+2. One from the pin you added labeled **P** to any available GPIO pin on the pi
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:193]:
```suggestion
2. One from the pin you added labeled **P** to any available GPIO pin on the Pi
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 24]
```diff
@@ -0,0 +1,304 @@
...
+3. One from terminal labeled **DIRExternal potentiometer** to any available GPIO pin on the pi
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:194]:
```suggestion
3. One from terminal labeled **DIRExternal potentiometer** to any available GPIO pin on the Pi
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 25]
```diff
@@ -0,0 +1,304 @@
...
+
+Finally, you'll want to connect the wires from the 12V to the 5v pi power converter to the battery.
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:196]:
```suggestion
Finally, you'll want to connect the wires from the 12V to the 5v Pi power converter to the battery.
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 26]
```diff
@@ -0,0 +1,304 @@
...
+
+At this point, you can plug in any extra components to the Pi (GPS and camera, if you are using them).
+If they are USB-based as per the parts list, this is easy.
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:199]:
```suggestion
If they are USB-based according to the parts list, this is easy.
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 27]
```diff
@@ -0,0 +1,304 @@
...
+
+## Bring your robot to life
+
+### Viam config overview
+
+We've now got all the hardware and wiring set up, so it is time to configure your robot to be controlled with software.
+You might be bracing yourself, as this part is usually much harder than what you just did.
+Thankfully, the Viam platform makes it relatively painless.
+
+First, let's install [viam-server](/installation/prepare/rpi-setup/) on your pi.
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:209]:
```suggestion
First, let's install [viam-server](/installation/prepare/rpi-setup/) on your Pi.
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 28]
```diff
@@ -0,0 +1,304 @@
...
+It should only take a few minutes and once you're done, you are ready to configure your robot.
+
+The Viam platform represents both individual hardware pieces and logical groupings of hardware as [components](/components/).
+Higher-level control is surfaced with [services](/services/).
+For this project, we'll configure:
+
+- One [board](/components/board/) component representing our Raspberry Pi
+- Two [motor](/components/motor/) components
+- One [base](/components/base/) component, referencing the two motors
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:218]:
```suggestion
- One [base](/components/base/) component, referencing the base connecting the two motors
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 29]
```diff
@@ -0,0 +1,304 @@
...
+In the [Viam app](http://app.viam.com/), navigate to **CONFIG** within the robot you created previously, and add a new component of type "board", model "pi".
+You can name it whatever you want, but you'll reference this name later.
+We'll call it "local" in this example.
+
+#### Motor component
+
+Next, create another component of type "motor", model "gpio".
+Call it "left" to represent the motor on the left side of your rover.
+Now we'll need to add some attributes.
+At minimum, we'll need to reference the board component we created in the last step by name, and the GPIO pins we connected from this motor to our pi.
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:244]:
```suggestion
At minimum, we'll need to reference the board component we created in the last step by name, and the GPIO pins we connected from this motor to our Pi.
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 30]
```diff
@@ -0,0 +1,304 @@
...
+Other attributes (Max RPM, PWM Freq) are [important for motor control](/components/motor/gpio#brushless-dc-motor) and can vary based on your specific hub motors.
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:245]: i feel like add some attributes you used if its necessary for this tutorial, saying "they are important but can vary based on the motor" makes it confusing

File: docs/tutorials/build-an-outdoor-rover.md [hunk 31]
```diff
@@ -0,0 +1,304 @@
...
+Finally, one of the two motors we are connecting will need the "Direction Flip" attribute set to "true".
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:246]: i would move this after you configure the other motor

File: docs/tutorials/build-an-outdoor-rover.md [hunk 32]
```diff
@@ -0,0 +1,304 @@
...
+It's ok if you are not sure which one, you can always switch them later.
+Regardless, once you've configured the "left" motor, repeat for the "right" - just make sure you're mapping the correct GPIO pins for each.
+
+<div class="td-max-width-on-larger-screens">
+  <img src="../img/outdoor-rover-boxbot/motor-setup.png" alt="Motor setup." title="Motor setup." />
+</div>
+
+At this point, you can try controlling your motors with the Viam App.
+Navigate to the control tab.
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:255]:
```suggestion
Navigate to the **CONTROL** tab.
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 33]
```diff
@@ -0,0 +1,304 @@
...
+If all is set up correctly, you'll see cards for your left and right motors on this page.
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:256]:
```suggestion
If all is set up correctly, you'll see panels for your left and right motors on this page.
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 34]
```diff
@@ -0,0 +1,304 @@
...
+Open one up, and if your rover is in a safe position (on the ground or propped up), try running one!
+
+Caution: Be aware that these are powerful motors, so you don't want to start at 100% power right away.
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:259]: i think this repeats below

File: docs/tutorials/build-an-outdoor-rover.md [hunk 35]
```diff
@@ -0,0 +1,304 @@
...
+
+## Take your Boxbot for a drive
+
+<div class="td-max-width-on-larger-screens">
+  <img src="../img/outdoor-rover-boxbot/completed-rover.jpg"  style="float:right;margin-left:1em;" alt="Completed rover with mounted solar panel." title="Completed rover with mounted solar panel." width="350" />
+</div>
+
+You can now have some real fun.
+Try driving your rover by keyboard.
+Drive your [robot progammatically](/tutorials/viam-rover/try-viam-sdk/), use a [color detector](/tutorials/viam-rover/try-viam-color-detection/) or a [machine learning detector](/services/vision/#detection) to interact with the environment.
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:292]:
```suggestion
Drive your [robot progammatically](/tutorials/viam-rover/try-viam-sdk/), use a [color detector](/tutorials/viam-rover/try-viam-color-detection/) or a [object detector](/services/vision/#detection) to interact with the environment.
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 36]
```diff
@@ -0,0 +1,304 @@
...
+Check out our [Python SDK documentation](https://python.viam.dev/) (or other SDK in the language of your choice) and start planning how you'll use your sturdy outdoor rover to do real things!
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:293]:
```suggestion
Check out our [Python SDK documentation](https://python.viam.dev/) (or another SDK in the language of your choice) and start planning how you'll use your sturdy outdoor rover to do real things!
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 37]
```diff
@@ -0,0 +1,304 @@
...
+
+If you're adding a camera, GPS, planning on using Viam's Navigation Service, or want to attach a solar panel and charge controller - this is all fairly plug and play with Viam and is covered in the [Viam documentation](/).
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:295]:
```suggestion
If you're adding a camera, or GPS, planning on using Viam's Navigation Service, or attaching a solar panel and charge controller - this is all fairly plug-and-play with Viam and is covered in the [Viam documentation](/).
```

File: docs/tutorials/build-an-outdoor-rover.md [hunk 38]
```diff
@@ -0,0 +1,304 @@
...
+
+<div class="td-max-width-on-larger-screens">
+  <img src="../img/outdoor-rover-boxbot/matt-done.png"  style="float:left;margin-right:1em;" alt="Matt working on project." title="Matt working on project." width="250" />
+</div>
+
+I had a lot of fun building this rover, and hope you did as well.
+I am now programming mine to help keep unwanted animal pests away from my garden, garbage, and egg-laying ducks using Viam's data management, vision, and navigation services.
```
Comment (mestcihazal) [inline docs/tutorials/build-an-outdoor-rover.md:302]:
```suggestion
I am now programming mine to help keep unwanted animals away from my garden, garbage, and egg-laying ducks using Viam's data management, vision, and navigation services.
```

**Review by mestcihazal — CHANGES_REQUESTED** [review]
changed a few works and shortened phrases, fixed typos, etc.

File: docs/tutorials/build-an-outdoor-rover.md (hunk 31 above)
Comment (mcvella) [reply docs/tutorials/build-an-outdoor-rover.md:247]: done

File: docs/tutorials/build-an-outdoor-rover.md (hunk 33 above)
Comment (mcvella) [reply docs/tutorials/build-an-outdoor-rover.md:256]: I think we call them cards?

File: docs/tutorials/build-an-outdoor-rover.md (hunk 35 above)
Comment (mcvella) [reply docs/tutorials/build-an-outdoor-rover.md:290]: machine learning is less specific and more of a searched-for term (but honest here)

File: docs/tutorials/build-an-outdoor-rover.md (hunk 37 above)
Comment (mcvella) [reply docs/tutorials/build-an-outdoor-rover.md:293]: I think the way it is written is actually more correct

File: docs/tutorials/build-an-outdoor-rover.md (hunk 35 above)
Comment (mestcihazal) [reply docs/tutorials/build-an-outdoor-rover.md:290]: machine learning detector just made it sound like you are detecting ml models. maybe "machine learning object detector"?

File: docs/tutorials/build-an-outdoor-rover.md (hunk 33 above)
Comment (mestcihazal) [reply docs/tutorials/build-an-outdoor-rover.md:256]: interesting didn't know

File: docs/tutorials/build-an-outdoor-rover.md (hunk 35 above)
Comment (mestcihazal) [reply docs/tutorials/build-an-outdoor-rover.md:290]: also, i just realized you have a typo on programmatically

**Review by mestcihazal — APPROVED** [review]
one typo, otherwise nits and then lgtm

**Pull request merged**